daemon/snap.go

```diff
@@ -240,6 +240,10 @@ func mapLocal(about aboutSnap, sd clientutil.StatusDecorator) *client.Snap {
 		result.GatingHold = &about.gatingHold
 	}
 
+	if snapst.RefreshInhibitedTime != nil {
+		result.PendingUpdate = true
+	}
```

Comment:
This could be simplified to:
```suggestion
	result.PendingUpdate = snapst.RefreshInhibitedTime != nil
```